File: src/main/java/com/practica/crean/arquitecture/core/usecases/CrearEstudiante.java
package com.practica.crean.arquitecture.core.usecases;

import com.practica.crean.arquitecture.core.domain.Estudiante;
import com.practica.crean.arquitecture.infraestructure.delivery.dto.EstudianteRequest;

public interface CrearEstudiante {

    Estudiante execute(EstudianteRequest estudiante);

}

File: src/main/kotlin/com/practica/crean/arquitecture/core/usecases/CrearEstudiante.kt
package com.example.demo.core.usecases

import com.example.demo.core.domain.Estudiante
import com.example.demo.infraestructure.delivery.dto.EstudianteRequest

fun interface CrearEstudiante {

    fun exectute(request: EstudianteRequest): Estudiante

}